From: Ozkan Sezer <sezeroz@gmail.com>
Date: Mon, 12 Sep 2022 20:28:00 +0300
Subject: apply commit d0a3570300812bc81888e7a7eadb9311621dc9a7 to cmake side
 too.

(cherry-picked from commit 5b2884cb0203cc63bf9753f8b55ea4c6c6f19cfb)
Origin: upstream, 2.24.1, commit:782862b95c0c87e4f875a6ddb0f0467ad0ff7060
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b53699..2fc5567 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1380,6 +1380,7 @@ elseif(UNIX AND NOT APPLE AND NOT RISCOS AND NOT HAIKU)
       check_c_source_compiles("
           #include <linux/kd.h>
           #include <linux/keyboard.h>
+          #include <sys/ioctl.h>
           int main(int argc, char **argv) {
               struct kbentry kbe;
               kbe.kb_table = KG_CTRL;
